src/zmi/styles/resources/zmi_base.js

```diff
@@ -31,7 +32,6 @@ function addItem( elm, base_url, zmi_dialog='modal') {
 			'manage_addRegistry',
 			'manage_addUserFolder',
 			'manage_addErrorLog',
-			'manage_addVirtualHostMonster',
```

Comment:
Are you planning to create a ticket to shorten this list after the PR is merged?

Reply:
Not "a ticket" but a ticket in each of the affected products' issue tracker that we control. Some we don't.